packages/core/src/crypto/__tests__/Jwk.test.ts

```diff
@@ -0,0 +1,124 @@
+import type { Bls12381g1Jwk, Bls12381g2Jwk, Ed25519Jwk, P256Jwk, P384Jwk, P521Jwk, X25519Jwk } from '../JwkTypes'
+
+import { getJwkFromKey, getKeyDataFromJwk } from '../Jwk'
+import { Key } from '../Key'
+import { KeyType } from '../KeyType'
+
+describe('jwk', () => {
```

Comment:
Are these some fixtures that are provided by the JWK specification or maybe generated by openssl or something? If so, ignore this :) otherwise I think it would be very good to add those as using the public from a jwk to verify a signature, but the key is incorrect would lead to very difficult debugging.